core/src/main/java/org/elasticsearch/search/sort/SortOrder.java

```diff
@@ -50,11 +50,8 @@ public String toString() {
             return "desc";
         }
     };
-    
-    private static final SortOrder PROTOTYPE = ASC;
 
-    @Override
-    public SortOrder readFrom(StreamInput in) throws IOException {
+    public static SortOrder readOrderFrom(StreamInput in) throws IOException {
```

Comment:
go for it, you don't break anything, it's a new method added in master only anyways